x-pack/plugins/snapshot_restore/__jest__/client_integration/policy_add.test.ts

```diff
@@ -210,6 +215,130 @@ describe('<PolicyAdd />', () => {
       });
     });
 
+    describe('feature states', () => {
+      beforeEach(async () => {
+        const { actions, form, component } = testBed;
+
+        // Complete step 1
+        form.setInputValue('nameInput', POLICY_NAME);
+        form.setInputValue('snapshotNameInput', SNAPSHOT_NAME);
+        actions.clickNextButton();
+
+        // Complete step 2
+        await act(async () => {
+          await nextTick();
```

Comment:
just a nit: Would it be possible to not use `nextTick` in these tests? Feel free to leave it for a follow up PR. I'll check if we have an issue for removing `nextTick` from `jest_helpers` and create one if not.

Reply:
Seems this also seems to be a thing for CCR and IM as well